test/e2e/components/vl-map-modify-action.js

```diff
@@ -0,0 +1,16 @@
+const VlMapAction = require('./vl-map-action');
+const VlMapFeaturesLayer = require('./vl-map-features-layer');
+
+
+class VlMapModifyAction extends VlMapAction {
+  async getFeaturesLayer() {
+    return new VlMapFeaturesLayer(this.driver, await this.driver.executeScript('return arguments[0].parentElement', this));
+  }
```

Comment:
Potentieel gevaar wanneer er andere elementen zouden tussenzitten. Voorlopig lijkt deze implementatie goed genoeg en lossen we dat probleem maar op als het zich stelt.